### ISSUE TITLE
Temporarily add  T1 tokens to `AirDropCurrencyId` for IAE and multicurrency initial allocations

### ISSUE THREAD

**Issue by balqaasem** [setheum/setheum#667]
```
                // VVV remove after IAE (Initial Airdrop Offering)
                // vvv AND MultiCurrency INITIAL ALLOCATIONS vvv
                "SETM",
                "SERP",
                "DNAR"
```

**Comment by balqaasem** [comment]
```
"AirDropCurrencyId": {
            "_enum": [
                "SETR",
                "SETUSD",
                // VVV remove after IAE (Initial Airdrop Offering)
                // vvv AND MultiCurrency INITIAL ALLOCATIONS vvv
                "SETM",
                "SERP",
                "DNAR"
            ]
        },
```

**Comment by balqaasem** [comment]
closes in On release #668
